core/templates/services/UpgradedServices.ts

```diff
@@ -85,6 +85,8 @@ import { ClassifierObjectFactory } from
   'domain/classifier/ClassifierObjectFactory';
 import { ClassroomBackendApiService } from
   'domain/classroom/classroom-backend-api.service';
+import { ClassroomDataObjectFactory } from
```

Comment:
Could you please add this to oppia-angular-root-component as well?

[Reference] It is mentioned under point 13 here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

Reply:
Done